Makefile.toml

```
@@ -0,0 +1,67 @@
+[config]
+skip_core_tasks = true
+default_to_workspace = false
+
+[tasks.test]
+cwd = "./build/"
+command = "make"
+args = [ "test" ]
+
+[tasks.build]
+dependencies = [ "cmake" ]
+command = "cmake"
+args = [ "--build", "build" ]
+
+[tasks.cmake]
+command = "cmake"
+args = [
+    "-S",
+    ".",
+    "-B",
+    "build",
+    "-DBUILD_TESTING=ON",
+]
+
+[tasks.lint-lib]
+cwd = "./include/"
+command = "poac"
+args = [ "lint" ]
+
+[tasks.lint-test]
+cwd = "./test/"
+command = "poac"
+args = [ "lint" ]
+
+[tasks.lint]
+dependencies = [ "lint-lib", "lint-test" ]
+
+[tasks.fmt-lib]
+install_crate = "fd-find"
+cwd = "./include/"
+command = "fd"
+args = [
+    "-e", "hpp",
+    "-e", "cpp",
+    "-x", "clang-format", "-i"
+]
+
+[tasks.fmt-test]
+install_crate = "fd-find"
+cwd = "./test/"
+command = "fd"
+args = [
+    "-e", "hpp",
+    "-e", "cpp",
+    "-x", "clang-format", "-i"
+]
+
+[tasks.fmt]
+dependencies = [ "fmt-lib", "fmt-test" ]
+ 
+[tasks.pretty]
+dependencies = [
+    "fmt",
+    "lint-lib",
```

```suggestion
    "lint",
```
ですかね？poac lintは--excludeサポートしているので、`poac lint --exclude Catch2 --exclude mitama-utest-utilities`でいけると思います。

---

いけました！